[DOCS-67] SDK overview (#573)
streamr-dev/streamr-platform: app/src/newdocs/content/SDKs.mdx

<ScrollableAnchor id="sdks-overview"><div>

## Overview

> There are both official and unofficial SDKs (Software Development Kits) available for building software that interacts

Reviewer: ```suggestion
There are both official and unofficial Software Development Kits (SDKs) available for building software that interacts
```

with the Streamr platform. Using an SDK provides an effortless and reliable way to use Streamr from your software

> projects. An SDK hides the intricacies involved of interacting with Streamr behind an intuitive library API, allowing a

Reviewer: ```suggestion
projects. The SDKs take care of the small details required to interact with Streamr through intuitive API libraries, allowing a
```

developer to focus on their task at hand.

Currently, we provide official SDKs for Javascript (browser and Node.js) and Java. There is also an experimental SDK for
Python in the works. The range of features each of these SDKs provide may vary; be sure to check the documentation of

> a specific SDK for more information.

Reviewer 2: A  feature matrix would be nice here. Here's a starting point, feel free to improve:

|                            | [JS](https://github.com/streamr-dev/streamr-client-javascript) | [Java](https://github.com/streamr-dev/streamr-client-java) | [Python](https://github.com/streamr-dev/streamr-client-python) |
| --                         | -- | --   | --     |
| Publish                    | ✅ | ✅   | ✅     |
| Subscribe                  | ✅ | ✅   | ✅     |
| Managing Streams           | ✅ | ✅   | ✅     |
| API Key Authentication     | ✅ | ✅   | ✅     |
| Ethereum Authentication    | ✅ | ✅   | ❌     |
| Duplicate Detection        | ✅ | ✅   | ✅     |
| Gap Filling                | ✅ | ✅   | ✅     |
| Message Signing            | ✅ | ✅   | ❌     |
| Message Verification       | ✅ | ✅   | ❌     |
| End-to-end Encryption      | ✅ | ✅   | ❌     |
| Requesting Historical Data | ✅ | ✅   | ❌     |

> a specific SDK for more information.

Reviewer: ```suggestion
the specific SDK for more information.
```

If your programming language or software environment of choice does not have an SDK available, you can still interact

> with Streamr using our [JSON-based HTTP API](api).

Reviewer: ```suggestion
with Streamr using our <Link to={links.newdocs.api}>JSON-based HTTP API</Link> .
```

Reviewer: this style of `<Link>` is better for internal redirects because it uses React Router (no page refresh).
